lib/src/command/get.dart

```diff
@@ -78,6 +129,12 @@ class GetCommand extends PubCommand {
       enforceLockfile: argResults['enforce-lockfile'],
     );
 
+    if (!argResults['dry-run'] && shouldRunPostGetHook()) {
+      log.message('Running post get hook...');
+      final exitCode = await runPostGetHook();
```

Comment:
I think this belongs at the end of Entrypoint.acquireDependencies. It has to run for everything that makes a resolution (get, upgrade, downgrade, add, remove)

Reply:
Entrypoint.acquireDependencies also has logic for determining if there where changes - and only in that case should the hook run.

Reply:
Sounds good!